DynamoDB/StandaloneFSharp/Program.fs

```diff
@@ -0,0 +1,64 @@
+﻿open Amazon.DynamoDBv2
+open Amazon.DynamoDBv2.DocumentModel
+open Amazon.DynamoDBv2.Model
+open Amazon.Runtime
+open RealtyShares
+open RealtyShares.DynamoDb
+open RealtyShares.DynamoEventStore
+
+[<EntryPoint>]
+let main argv = 
+    // Use credentials from environmental variables, and "N. California",
+    // which is `Amazon.RegionEndpoint.USWest1`, is our default. -- D.K.
+    let credentials = new EnvironmentVariablesAWSCredentials()
+    let client = new AmazonDynamoDBClient(credentials, Amazon.RegionEndpoint.USWest1)
+    let tableName = "StandaloneFSharp-demo-table"
+
+    async {
+        printfn "Creating table, %s..." tableName
+        let! res = createTable client tableName
```

Comment:
Btw, if the test is terminated in the middle, it won't start again until the table is manually deleted :)